src/utils/apiPaths.ts

```diff
@@ -45,8 +45,9 @@ export const makeLanguagesUrl = (language: string): string =>
  * @returns {string}
  */
 export const makeRecitersUrl = (): string => makeUrl('/audio/reciters');
-export const makeAudioFilesUrl = (reciterId: number, chapter: number) =>
-  makeUrl(`/audio/reciters/${reciterId}`, { chapter });
+
+export const makeAudioFilesUrl = (reciterId: number, chapter: number, segments: boolean) =>
```

Comment:
So we are actually reusing the same API path.

We might need to refactor the data flow to get audioFile and timestamps data. So they can be fetched at once.